scriptmodules/supplementary/wifi.sh

```diff
@@ -16,7 +16,7 @@ rp_module_flags="!x11"
 
 function remove_wifi() {
     sed -i '/RETROPIE CONFIG START/,/RETROPIE CONFIG END/d' "/etc/wpa_supplicant/wpa_supplicant.conf"
-    ifdown wlan0 &>/dev/null
+    ifdown wlan0 &>/dev/null || ip link set wlan0 down &>/dev/null
```

Comment:
if the second case works on jessie too, we can just use that rather than trying both ?

Reply:
also is &>/dev/null actually needed for the ip link set command ? might just be ok to redirect stderr.

Reply:
No, the second case doesn't work on jessie. 

On stretch, the wlan0 device is not enumerated in /etc/network/interfaces, and the dhcpcd service is supposed to automatically trigger wpa_supplicant as needed. On jessie, wlan0 is configured via /etc/network/interfaces and wpa_supplicant gets triggered via that, and simply bringing up the interface (or also restarting dhcpcd) is not enough to establish a connection, as wpa_supplicant doesn't trigger via dhcpcd.

Reply:
ok thanks.

Reply:
actually, it would make sense then do have a function which does the jessie or stretch method and call that - would de-duplicate the code a bit.

Reply:
I was considering that, but figured that it's potentially not a problem isolated to Raspbian jessie -> stretch. Other boards may benefit from trying both the regular ifup and then trying the fallback.

Reply:
it can still be in a function to de-duplicate.